containers/forwarder/Dockerfile

```diff
@@ -4,10 +4,10 @@ ADD containers/forwarder/bin/run.sh /
 ADD .splunk-version /
```

Comment:
Drive-by note: since #99 this won't even run unless `.splunk-version[-hash]` has changed.